x/auth/ante/sigverify.go

```diff
@@ -323,6 +323,18 @@ func (svd SigVerificationDecorator) consumeSignatureGas(
 // - the pub key is on the curve.
 // - verify sig
 func (svd SigVerificationDecorator) verifySig(ctx sdk.Context, simulate bool, tx sdk.Tx, acc sdk.AccountI, sig signing.SignatureV2) error {
+	unorderedTx, ok := tx.(sdk.TxWithUnordered)
```

Comment:
Can we update the godoc of  this function to reflect the new logic?

This was never changed when these updates were made

Reply:
gave the entirety of the comment a fresh set of paint. [update comment](https://github.com/cosmos/cosmos-sdk/pull/23734/commits/86add1400d493ad196d4a067c55ac3d8577c1f62)